source/blog/2018-03-06-release-notes.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: 🌷 March 2018 Release Notes - Version 18.3.0
+published: true
+post_author:
+  display_name: Stefan Lance
+  twitter: @stefan_lance
+tags: Releases, Finagle, Finatra, Util, Scrooge, TwitterServer
+---
+
+Our March releases have arrived!
+
+[Finagle 18.3.0][finagle], [Finatra 18.3.0][finatra], [Scrooge 18.3.0][scrooge], [TwitterServer 18.3.0][twitterserver], and [Util 18.3.0][util].
+
+### [Finagle](https://github.com/twitter/finagle/) ###
+
+New Features
+--------------
```

Comment:
I think the '_' need to align with the subtitle for markdown, actually we may not need the underscores here.